lighthouse-core/runner.js

```diff
@@ -131,10 +140,11 @@ class Runner {
         configSettings: settings,
         categories,
         categoryGroups: opts.config.groups || undefined,
-        timing: {total: Date.now() - startTime},
+        timing: artifacts.Timing,
       };
 
       const report = generateReport(lhr, settings.output);
+      log.timeEnd(status);
```

Comment:
seems like we're double `timeEnd`ing this one?

Reply:
thx fixed.